packages/navie/src/commands/review-command.ts

```diff
@@ -208,6 +208,16 @@ ${userPrompt}
         { locations }
       );
     }
+
+    // For backwards compatibility, include the code selections which have been sent
+    // without a location.
+    pinnedItems.filter(UserContext.isCodeSelectionItem).forEach((cs) =>
+      pinnedItemLookup.push({
+        type: ContextV2.ContextItemType.CodeSelection,
+        content: cs.content,
+      })
+    );
```

Comment:
Small nitpick — `forEach` is great for when you already have a named function to iterate with, but with a lambda like that `for ... of` is IMO much more readable (not to mention about 20× faster). Or do something like this:
```suggestion
    pinnedItemLookup.push(
      ...pinnedItems
        .filter(UserContext.isCodeSelectionItem)
        .map((i) => ({ ...i, type: ContextV2.ContextItemType.CodeSelection }))
    );
```

Reply:
(BTW, it's weird that we have to retype `type` like that even though the value is the same. `ContextV2.ContextItemType` should really have been a union type, not an enum.)